middlewares/authentication.js

```diff
@@ -34,4 +34,24 @@ const verifyToken = (req, res, next) => {
   });
 };
 
-export default verifyToken;
+const getUserId = (req, res, next) => {
```

Comment:
Why is this middleware important?

Reply:
We want to be able to get a user's `ID` in order to check if he's bookmarked a particular article when fetching it.
The `verifyToken` middleware won't serve the purpose because if an unauthenticated user tries to access the route, it would return an `unauthorized` error.

Reply:
👍